library/src/main/java/com/cube/storm/UiSettings.java

```diff
@@ -591,6 +598,19 @@ public Builder youtubeApiKey(@Nullable String youtubeApiKey)
 			isYouTubeAPIKeyInitialised = true;
 			return this;
 		}
+		
+		/**
+		 * Sets the app to display chevrons on StandardListItemView if the link matches the correct criteria
+		 *
+		 * If not set, StandardListItemView will not display with chevrons
+		 *
+		 * @return The {@link com.cube.storm.UiSettings.Builder} instance for chaining
+		 */
+		public Builder chevronSpec(ChevronSpec spec)
```

Comment:
Not one specifically for this PR, but when using class types it's good to annotate with nullability annotations if possible. This helps with Kotlin integration too presumably.

Reply:
Good point, will get to doing this.